### PR TITLE
update to install OpenVINO and Docker on the VCAC-A

### PR DESCRIPTION
**Pull request by QuPengfei**

### CODE REVIEW

File: VCAC-A/README.md [hunk 1]
```diff
@@ -3,28 +3,29 @@ Intel VCAC-A is designed to accelerate analytics computation. This README descri
...
-Please follow the [Software Installation Guide, Section 2](https://cdrdv2.intel.com/v1/dl/getContent/611894) to build and configure the software packages for the host and the VCAC-A, with the following additional steps:    
+Please follow the [Software Installation Guide, Section 2.0-Section 2.2.4](https://cdrdv2.intel.com/v1/dl/getContent/611894) to build and configure the software packages for the host and the VCAC-A, with the following additional steps:    
 
 - **Install Docker Engine on Host and VCAC-A**
 
-Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.    
+Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.
 
-- **Install Intel OpenVINO on VCAC-A**
+Here is the script to install Docker on the VCAC-A:
```
Comment (jhou5) [inline VCAC-A/README.md]: Here are the scripts to ...

Comment (xwu2git) [reply VCAC-A/README.md]: (1)  Please rename the scripts to be more uniformed:
setup-swarm.sh
upload-images.sh
upload-image1.sh
docker-run.sh
container-launcher.sh

(2) In all scripts, please add  $(env | grep -E '_(proxy)=' | sed 's/^/--build-arg /') to docker build, and --rm $(env | grep -E '_(proxy)=' | sed 's/^/-e /') to dock run commands.

(3) Some of the scripts use PROXY and some use proxy. Please use proxy uniformly.

Comment (QuPengfei) [reply VCAC-A/README.md]: update as the comments. Please review again.

File: VCAC-A/script/setup-access.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+cat /dev/zero | ssh-keygen -q -N ""
```
Comment (1480c1) [inline VCAC-A/script/setup-access.sh]: `ssh-keygen -q -N < /dev/zero` useless use of cat

File: VCAC-A/script/setup-docker.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+install_docker_engine()
+{
+    http_proxy="$1"
+    https_proxy="$2"
+    no_proxy="$3"
+    dnss="$4"
+
+    check_docker=`docker images`
+    if [ $? != 0 ];then
```
Comment (1480c1) [inline VCAC-A/script/setup-docker.sh]: `if ! check_docker=$(docker images); then`

File: VCAC-A/script/setup-docker.sh [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+case "$0" in 
+    *install*)
+        install_docker_engine "$@"
+        ;;
+    *setup*)
+        sudo yum -y install NetworkManager
+        systemctl start NetworkManager
+        http_proxy=${http_proxy:-${http_proxy}}
+        https_proxy=${https_proxy:-${https_proxy}}
+        no_proxy=${no_proxy:-${no_proxy}}
```
Comment (1480c1) [inline VCAC-A/script/setup-docker.sh]: These do not make sense, set no_proxy to (if no_proxy is empty or null, return no_proxy, else no_proxy)

File: VCAC-A/script/setup-hddl.sh [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+	ov_ver=`cat ov_ver.log`
+	mkdir -p /opt/intel/
+	tar zxf ${package_name}.tar.gz -C /opt/intel/
+	mv /opt/intel/openvino /opt/intel/$ov_ver
+        ln -s /opt/intel/$ov_ver /opt/intel/openvino
+        export LC_ALL=C
+        apt-get install -y cmake libelf-dev
+	cd /opt/intel/$ov_ver/deployment_tools/inference_engine/external/hddl/drivers/drv_ion 
+	make && make install
+	if [ $? != 0 ];then
```
Comment (1480c1) [inline VCAC-A/script/setup-hddl.sh]: `if ! { make && make install; }; then`

Comment (xwu2git) [reply VCAC-A/script/setup-hddl.sh]: $(env | grep -E '(proxy|REPO|VER)=' | sed 's/^/--build-arg /') should be added before ${DIR} in docker build.

File: VCAC-A/README.md [hunk 6]
```diff
@@ -3,28 +3,29 @@ Intel VCAC-A is designed to accelerate analytics computation. This README descri
...
 
 - **Install Docker Engine on Host and VCAC-A**
 
-Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.    
+Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.
 
-- **Install Intel OpenVINO on VCAC-A**
+Here are the scripts to install Docker on the VCAC-A:
+-  **```Setup Passwordless access```** : [setup-access.sh](./script/setup-access.sh)
+-  **```Install Docker or Docker CE on VCAC-A```**: [setup-docker.sh](./script/setup-docker.sh)
```
Comment (xwu2git) [inline VCAC-A/README.md]: Install docker-ce (setup-docker.sh). Alternatively, you can install docker-ee instead on the VCAC-A yourself.

File: VCAC-A/README.md [hunk 7]
```diff
@@ -3,28 +3,29 @@ Intel VCAC-A is designed to accelerate analytics computation. This README descri
...
+-  **```Install OpenVINO and start hddldaemon on VCAC-A```** : [setup-hddl.sh](./script/setup-hddl.sh)
 
-Login to VCAC-A and install the Intel OpenVINO software by downloading from this [link](https://software.intel.com/en-us/openvino-toolkit/choose-download). The package name should read ```l_openvino_toolkit_p_201x.x.xxx.tgz```. During installation, the only required component is ```Inference Engine```. 
-
-After installation, start the HDDL daemon as follows. 
+After installation, start the HDDL daemon manually as the following if hddldaemon does not start.
```
Comment (xwu2git) [inline VCAC-A/README.md]: ... as follows ...

File: VCAC-A/script/setup-docker.sh (hunk 4 above)
Comment (xwu2git) [reply VCAC-A/script/setup-docker.sh]: CAPS missing. They should be:

        http_proxy=${http_proxy:-${HTTP_PROXY}}
        https_proxy=${https_proxy:-${HTTPS_PROXY}}
        no_proxy=${no_proxy:-${NO_PROXY}}

File: VCAC-A/README.md (hunk 7 above)
Comment (xwu2git) [reply VCAC-A/README.md]: Actually, this section is not needed as setup-hddl.sh will auto-start hddldaemon. The user should not care about hddldaemon.

**Comment by xwu2git** [comment]
The following paragraph can be removed:

Install Docker Engine on Host and VCAC-A
Follow the instructions to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.

Here are the scripts to install Docker on the VCAC-A:

**Comment by xwu2git** [comment]
This step can be removed: Section 4: . Setup VCAC-A Passwordless Access as the above. This is already done as part of the setup. 

The script update_images.sh is not mentioned in the README.

**Comment by xwu2git** [comment]
Fix formatting to list See Also and the two samples on different lines. Now they look as follows:

See also AD Insertion Sample Smart City Sample

**Pull request merged**